### DISCUSSION TITLE
Github student?

### DISCUSSION THREAD

**Discussion by hardingjo** [USU-ITLS-x265-23-summer/class-repo#8] [Week 1 - Questions]
I've tried multiple methods of documentation for the student verification, but Github keeps denying with emails like this:
<img width="675" alt="Screenshot 2023-05-09 at 2 04 02 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132959701/769b8bbf-f357-4fd6-b00a-74a5954afd1c">
Anyone that has gotten through this step, what did you do?

**Comment by metasean** [comment]
duplicate of #1
